tsconfig.base.json

```
@@ -22,6 +22,7 @@
       "@openfeature/env-var-provider": ["libs/providers/env-var/src/index.ts"],
       "@openfeature/flagd-core": ["libs/shared/flagd-core/src/index.ts"],
       "@openfeature/flagd-provider": ["libs/providers/flagd/src/index.ts"],
+      "@openfeature/ofrep-shared": ["libs/shared/ofrep-core/src/index.ts"],
```

```suggestion
      "@openfeature/ofrep-core": ["libs/shared/ofrep-core/src/index.ts"],
```

This matches the file path. There are likely other places that reference `@openfeature/ofrep-shared`. Please make sure to change those too if you accept this proposal.

---

Just checked it, it was only here.
No code referenced this until now so this was the only place.